types/table/index.d.ts

```diff
@@ -1,50 +1,52 @@
 // Type definitions for Tabris.js 4.0
 // Project: https://github.com/gajus/table
 // Definitions by: Evan Shortiss <https://github.com/evanshortiss>
+//                 mrmlnc <https://github.com/mrmlnc>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export type borderType = 'honeywell' | 'norc' | 'ramac' | 'void';
+export type BorderType = 'honeywell' | 'norc' | 'ramac' | 'void';
```

Comment:
This would be a breaking change. I don't have an issue with it personally, but worth bearing in mind.

Reply:
Hm, I hadn't even thought of that. I try to find code: https://github.com/search?utf8=✓&q=%40types+table&type=Code

Well, I can change this back :)